soh/src/overlays/actors/ovl_En_Mag/z_en_mag.c

```diff
@@ -618,6 +619,32 @@ s16 GetCharArraySize(const char* str) {
     return length;
 }
 
+static char* EnMag_GetCopyrightTex() 
+{
+    uint32_t gameVersion = ResourceMgr_GetGameVersion(0);
+
+    switch (gameVersion) {
+        case OOT_PAL_11:
+            return "__OTR__objects/object_mag/gTitleCopyright1998Tex";
+        default:
+            return "__OTR__objects/object_mag/gTitleCopyright19982003Tex";
```

Comment:
Based on an investigation performed here https://github.com/HarbourMasters/Shipwright/pull/2819 and the linked issue inside.

We should be using the aligned asset value rather than a raw string in the DList commands. You may need to either manually add a new aligned value for this 1998 tex, or try extracting the headers with the 1.1 and gamecube together.